### PR TITLE
Add validation to Resume fields in information structure

### LINKED ISSUE #106: Add validation to Resume fields in information structure
Collection name: `"resume"`
Task description in linked Epic issue.
https://github.com/CodersCrew/CodersCard/issues/98#issue-725735917

### PR DESCRIPTION
**Pull request by KlaraGajaszek**
close #106

### CODE REVIEW

File: static/admin/config.yml [hunk 1]
```diff
@@ -119,23 +119,23 @@ collections:
     editor:
       preview: false
     fields:
-      - { label: "Title", name: "resume-page-title", widget: "string" }
+      - { label: "Title", name: "resume-page-title", widget: "string",pattern: ['.{15}', "Must have at least 15 characters"] }
```
Comment (mecies) [inline static/admin/config.yml]: Please add a max length to the page title at 50 or 60 characters (preferably 60 like on the blog page)
source:
https://moz.com/learn/seo/title-tag#:~:text=Optimal%20title%20length,your%20titles%20to%20display%20properly.

File: static/admin/config.yml [hunk 2]
```diff
@@ -119,23 +119,23 @@ collections:
...
       - label: "Work experience"
         name: "work-experience"
         widget: "list"
         fields:
-          - { label: "Job title", name: "job-title", widget: "string" }
-          - { label: "Company name", name: "company-name", widget: "string" }
-          - { label: "Job description", name: "job-description", widget: "string" }
+          - { label: "Job title", name: "job-title", widget: "string",pattern: ['.{3,50}', "Must have 3-50 characters"] }
```
Comment (mecies) [inline static/admin/config.yml]:
```suggestion
          - { label: "Job title", name: "job-title", widget: "string", pattern: ['.{3,50}', "Must have 3-50 characters"] }
```
please add a trailing space before pattern and repeat in every occurrence 😄

File: static/admin/config.yml [hunk 3]
```diff
@@ -119,23 +119,23 @@ collections:
...
       - label: "Education"
         name: "education"
         widget: "list"
         fields:
-          - { label: "School/University name", name: "school-name", widget: "string" }
-          - { label: "Course/Degree", name: "course", widget: "string" }
-          - { label: "Education description", name: "education-description", widget: "string" }
+          - { label: "School/University name", name: "school-name", widget: "string",pattern: ['.{9}', "Must have at least 9 characters"] }
+          - { label: "Course/Degree", name: "course", widget: "string",pattern: ['.{5,100}', "Must have 5-100 characters"] }
+          - { label: "Education description", name: "education-description", widget: "string",pattern: ['.{80,4000}', "Must have 80-4000 characters"] }
```
Comment (mecies) [inline static/admin/config.yml]: 4000 characters seems a bit long for an education description, have you checked whether it looks good?
If it looks good on desktop, tablet, and mobile then fine 😄

**Review by mecies — CHANGES_REQUESTED** [review]
Minor comments, please address, other than that, good job 👏

File: static/admin/config.yml (hunk 3 above)
Comment (KlaraGajaszek) [reply static/admin/config.yml]: Yea 400 not 4000

**Review by mecies — APPROVED** [review]
Nice job, ready to merge ✅

**Pull request merged**